packages/server/src/auth/auth.controller.ts

```diff
@@ -27,6 +27,7 @@ export class SignupController {
   @Post()
   @UsePipes(new SignupTransformPipe())
   async signup(@Body() user: UserSignupDto): Promise<AccessToken> {
-    return await this.authService.signup({ ...user });
```

Comment:
Why not just `return this.authService.signup(user);`?

Reply:
Woops I has a console.log between that which is why I assigned to a variable. Forgot to change it back.

Also removed the deconstruct.

Thanks